.circleci/config.yml

```diff
@@ -156,6 +156,12 @@ workflows:
       - test-e2e-firefox-snaps:
           requires:
             - prep-build-test-flask
+      - test-e2e-chrome-blockaid:
+          requires:
+            - prep-build-test-flask
```

Comment:
The build cannot be **_exactly_** the same as flask, as we need to pass 2 envars in order to make ppom work:

    BLOCKAID_FILE_CDN
    BLOCKAID_PUBLIC_KEY

For the CDN link, we should use the test one (once confirmed is working).

Reply:
Valid point on including the valid envars, should do. However this activates blockaid UI (fallback response) until the descriptive message is confirmed to work.

Reply:
This has been asked in the channel. Waiting for a response on what should be the best solution

Reply:
prod envars are now added on the config file. We just need to pass the test cdn url on the package-json script